### PR TITLE
Rename noop -> bridge

### PR DESCRIPTION
**Pull request by ilayaperumalg**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -189,7 +189,7 @@
 		<!--TODO: Move transport related dependencies here pending https://jira.spring.io/browse/XD-3337  -->
 		<dependency>
 			<groupId>org.springframework.cloud</groupId>
-			<artifactId>spring-cloud-stream-binder-redis</artifactId>
+			<artifactId>spring-cloud-stream-binder-rabbit</artifactId>
```
Comment (ericbottard) [inline pom.xml:192]: Do we want to commit this now ?

Comment (ericbottard) [inline pom.xml:192]: Also note that line 196 still has redis-starter.
I'm merging w/o the switch to rabbit, which deserves its own PR/commit I believe

Comment (ilayaperumalg) [inline pom.xml:192]: No, sorry.  It was on my local only as I was using `rabbit` binder but added that to this commit by mistake. yeah, please merge without rabbit binder. thank you.

**Comment by ericbottard** [comment]
Merged as https://github.com/spring-cloud/spring-cloud-stream-modules/commit/c00423ee9edc4886f88552f2590c37a0a7221ebd

Also did https://github.com/spring-cloud/spring-cloud-dataflow/commit/37f69e1e18262672ec97ffff68a49c82696f68f4

File: bridge-processor/src/main/java/org/springframework/cloud/stream/module/bridge/BridgeProcessor.java [hunk 2]
```diff
@@ -22,12 +22,12 @@
...
+ * A Processor application that connects input and output channels.
  *
  * @author Glenn Renfro
  */
 @EnableBinding(Processor.class)
-public class NoOpProcessor {
+public class BridgeProcessor {
 
 	@Transformer(inputChannel = Processor.INPUT, outputChannel = Processor.OUTPUT)
 	public Object transform(Message<?> message) {
```
Comment (artembilan) [inline bridge-processor/src/main/java/org/springframework/cloud/stream/module/bridge/BridgeProcessor.java:33]: I may be a bit late in this game, but usage of `BridgeHandler` would really be better. As for this processor premise as well as by the performance avoiding SpEL method evaluations.

Comment (markfisher) [inline bridge-processor/src/main/java/org/springframework/cloud/stream/module/bridge/BridgeProcessor.java:33]: +1
